solution/ui/regulations/js/src/components/RelatedRuleList.vue

```diff
@@ -27,7 +27,7 @@
             </template>
         </template>
         <collapse-button
-            v-if="showMoreNeeded && rulesCount > 10"
```

Comment:
There definitely is a bug in this component though.  It should mirror the behavior of the collapse buttons in `SupplementalContentList` below.

So that would mean having the `v-if` directive condition in this first `collapse-button` to be only `showMoreNeeded`:

`v-if="showMoreNeeded"`

and the `v-if` directive of the second `collapse-button` below on line 84 be:

`v-if="showMoreNeeded && rulesCount > 10"`

This will put both components in sync where they only show the "middle" Show More button if there are between five and ten items, and will show both the "middle" Show More button and the Show More button at the bottom of the list of items if there are ten or more items.